client/src/nl/sensorlab/videowall/animation/baseanimations/sorting/BubbleSort.java

```diff
@@ -0,0 +1,18 @@
+package nl.sensorlab.videowall.animation.baseanimations.sorting;
+
+public class BubbleSort extends Sort {
+
+	protected int current;
+
+	public BubbleSort(int[] arr) {
+		super(arr);
+		current = 0;
+		num = 1;
+	}
+
+	public void sortStep() {
+		if (current + 1 == valuesArray.length) current = 0;
```

Comment:
Please use accolades (curly braces) around conditional code blocks, and on separate lines. In multiple Sorting classes this pops up.